src/Raven.Client/Http/RequestExecutor.cs

```diff
@@ -238,29 +276,22 @@ public void Execute<TResult>(RavenCommand<TResult> command, JsonOperationContext
                     else
                     {
                         response = await command.SendAsync(client, request, token).ConfigureAwait(false);
+                        if (response.Headers.Contains(Constants.Headers.ForceTopologyUpdate))
```

Comment:
Under what conditions will the server send it? --> we have talked that the server will send it when senator is changed or when there are any changes to topology that come from the server-side